sequence_processing_pipeline/tests/test_Pipeline.py

```diff
@@ -199,6 +199,31 @@ def test_sample_sheet_validation(self):
         self.assertEqual(str(msgs[0]), 'ErrorMessage: A sample already exists '
                                        'with lane 1 and sample-id EP479894B04')
 
+    def test_generate_sample_information_files(self):
+        # test sample-information-file generation.
+        pipeline = Pipeline(self.good_config_file, self.good_run_id,
+                            self.good_output_file_path, 'my_qiita_id',
+                            None)
+
+        # assume a validated sample-sheet. Also,
+        # generate_sample_information_files calls validate() itself to ensure
+        # proper operation.
+        paths = partial(pipeline.generate_sample_information_files)
+        paths = paths(self.good_sample_sheet_path)
+
+        # for each file generated, first check that it has the proper header.
+        # then check that all subsequent lines in the file follow the pattern
+        # of entries we expect.
+        for path in paths:
+            with open(path, 'r') as f:
+                lines = f.readlines()
+                lines = [x.strip() for x in lines]
+                self.assertEqual(lines.pop(0),
+                                 'sample_name	host_subject_id	description')
+                for line in lines:
+                    results = re.search('^BLANK.*?\tBLANK.*?\tBLANK.*?$', line)
```

Comment:
Can you add and check that the correct values; as provided by Gail via email are added vs. placeholders?

Reply:
Ah, I think you mean the full_header as opposed to using only three of the columns (short_header)? I can do that.

Reply:
However as I mentioned earlier, only three or maybe four of the column values in Gail's example can be determined based on the metadata contained in the sample-sheet, from what I can see. For the rest I can introduce a blank or a placeholder.

Reply:
'#' is currently the placeholder value.

Reply:
Placeholders replaced with true defaults.